packages/selleo-design-core/src/parser.tsx

```diff
@@ -28,3 +28,18 @@ export const parseComponent = (html: string) => {
     ReactString: ReactStringResult,
   };
 };
+
+export const parseComponentWithoutAstro = (
+  Component: (args: typeof props) => h.JSX.Element,
+  props: { [key: string]: unknown }
```

Comment:
do you have the `props` available in the Compoent type declaration?

Reply:
Yes, Component prop has types from a props types